app/assets/javascripts/controllers/middleware_server/middleware_server_controller.js

```diff
@@ -120,15 +120,15 @@ function MwServerControllerFactory($scope, miqService, isGroupDeployment) {
 
   $scope.resetJdbcDriverForm = function () {
     $scope.jdbcDriverModel = {};
-    if(angular.element('#server_id').val()){
-        $scope.jdbcDriverModel.serverIdKey = 'id';
-        $scope.jdbcDriverModel.serverId  = angular.element('#server_id').val()
-    }else{
-        $scope.jdbcDriverModel.serverId  = ManageIQ.gridChecks.join(',')
-        $scope.jdbcDriverModel.serverIdKey = 'miq_grid_checks';
+    var server_id = angular.element('#server_id').val();
```

Comment:
Same applies to here, `_` is not good, since we use camel case in most of JS variables.